src/WingetCreateCLI/Commands/UpdateCommand.cs

```diff
@@ -42,8 +42,8 @@ public static IEnumerable<Example> Examples
             {
                 yield return new Example(Resources.Example_UpdateCommand_SearchAndUpdateVersionAndInstallerURL, new UpdateCommand { Id = "<PackageIdentifier>", InstallerUrls = new string[] { "<InstallerUrl1>", "<InstallerUrl2>" }, Version = "<Version>" });
                 yield return new Example(Resources.Example_UpdateCommand_SaveAndPublish, new UpdateCommand { Id = "<PackageIdentifier>", Version = "<Version>", OutputDir = "<OutputDirectory>", GitHubToken = "<GitHubPersonalAccessToken>" });
-                yield return new Example(Resources.Example_UpdateCommand_OverrideArchitecture, new UpdateCommand { Id = "<PackageIdentifier>", InstallerUrls = new string[] { "<InstallerUrl1>|<InstallerArchitecture>" }, Version = "<Version>" });
-                yield return new Example(Resources.Example_UpdateCommand_OverrideScope, new UpdateCommand { Id = "<PackageIdentifier>", InstallerUrls = new string[] { "<InstallerUrl1>|<InstallerScope>" }, Version = "<Version>" });
+                yield return new Example(Resources.Example_UpdateCommand_OverrideArchitecture, new UpdateCommand { Id = "<PackageIdentifier>", InstallerUrls = new string[] { "“<InstallerUrl1>|<InstallerArchitecture>”" }, Version = "<Version>" });

```

Comment:
```suggestion
                yield return new Example(Resources.Example_UpdateCommand_OverrideArchitecture, new UpdateCommand { Id = "<PackageIdentifier>", InstallerUrls = new string[] { "'<InstallerUrl1>|<InstallerArchitecture>'" }, Version = "<Version>" });

```

I think we can get by with using single quote. I'm sure using a unicode character is fine but probably better to stick with ASCII for now just to be safe. Don't forget to update the docs to reflect this change.

Reply:
That was the first thing I tried.....but....I could swear it wasn't working last time i tried it 😴 smh, how this works perfectly now is beyond me